homework/App.js

```diff
@@ -55,7 +71,10 @@ class App {
       const leftDiv = Util.createAndAppend('div', container);
       const rightDiv = Util.createAndAppend('div', container);
 
-      const contributorList = Util.createAndAppend('ul', rightDiv);
+      const contributorList = Util.createAndAppend('ul', rightDiv, {
+        text: 'Contributors',
+        class: 'rightDiv',
```

Comment:
`kebab-case` instead of `camelCase` for class attributes please 🙂

Reply:
Done !